CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 All notable changes to [Solang](https://github.com/hyperledger/solang/)
 will be documented here.
 
+## Unreleased
+
+### Fixed
+- Resolving import paths now matches solc more closely, and only resolves relative
+  paths when specified as `./foo` or `../foo`. [seanyoung](https://github.com/seanyoung)
```

Comment:
This is a breaking change, isn't it? Perhaps we should mark it with `**BREAKING**`

Reply:
Yes, it is. Added.